Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginUIFactory.swift

```diff
@@ -6,6 +6,7 @@ struct SimplifiedLoginUIFactory {
     static func buildViewController(client: Client,
                                     userContext: UserContextFromTokenResponse,
                                     userProfileResponse: UserProfileResponse,
+                                    visibleClientName: String? = nil,
```

Comment:
Just thinking about this one. Should we ever allow visibleClientName to be nil? 🤔

Reply:
In this place we can, because if it's not present we will use CFBundleDisplayName

Reply:
Now the empty string is a placeholder but we can always change it to sth like "configure_me" :P

Reply:
Or we can print warning in logs. What do you think?

Reply:
Hmm perhaps not the "configure_me" option. 
Maybe guard for it? 🤔  For example. if CFBundleDisplayName is empty string and no clientName provided by brand then throw a SimplifiedLoginError

Reply:
Done, you are right. Looks much better